community/concepts/grain.md

```diff
@@ -0,0 +1,73 @@
+### Grain
+
+SourceCred is both a social algorithm for assigning [cred](concepts/cred.md)
+scores, and a cryptoeconomic system for rewarding a project's contributors
+and sponsors. That cryptoeconomic system revolves around _grain_, a cred-based
+cryptoasset.
+
+The key properties of grain include:
+
+- **Grain is project-specific.**
+
+It's not a universal token; every project that is using SourceCred
+creates its own independent token. They can call it something other
+than grain (e.g. "Seeds") if they want.
+
+- **Grain is harvested by contributors, based on cred scores.**
+
+Grain is created during "harvests". In a harvest, a project mints new grain and
```

Comment:
Periods go on the inside of parentheses.

Reply:
Which parentheses?

Reply:
Oops, meant quotes, e.g., instead of "harvests". we'd want "harvests."

Reply:
I see. For me, conceptually, I think the period should be outside the quote--becuase the quotes are a modifier on the word "harvests", but not a modifier on the sentence as a whole. However, the internet agrees with you https://www.grammarly.com/blog/quotation-marks/ so I changed it.